cdhweb/pages/blocks/article_index_block.py

```diff
@@ -0,0 +1,66 @@
+from springkit.blocks.headings import HeadingBlock
+from springkit.blocks.jumplinks import JumplinkMixin
+from wagtail import blocks
+
+from cdhweb.pages.blocks.link import InternalPageLinkBlock
+
+
+class ArticleTileBlock(JumplinkMixin):
+    """
+    CMS controlled Article Tile block
+    """
+
+    heading = HeadingBlock(required=False, help_text="Heading for this tile block")
+
+    description = blocks.CharBlock(
+        help_text="Description for this tile block",
+        label="Description",
+        required=False,
+    )
+
+    landing_page = blocks.PageChooserBlock(
+        required=True,
+        page_type=["blog.BlogLandingPage"],
+        help_text=(
+            "Select the Blog Landing Page whose child pages you want to show as "
+            "tiles in this block."
+        ),
+    )
+
+    max_articles = blocks.ChoiceBlock(
+        choices=[
+            (3, "3"),
+            (6, "6"),
+            (9, "9"),
+        ],
+        default=3,
+        icon="placeholder",
+        help_text="Define the maximum number of tiles to show in this group.",
+    )
+
+    see_more_link = InternalPageLinkBlock(help_text="'See more' link", required=False)
```

Comment:
Would this ever be anything other than the landing page? Maybe we could just make this control the text?